packages/app/src/device/modules/device.ts

```diff
@@ -73,22 +73,19 @@ export class Device {
     }
   }
 
-  public async disconnect(): Promise<ResultObject<boolean>> {
-    const response = await this.strategy.disconnect()
-
-    this.unmountDeviceListeners()
-
-    if (response) {
-      return Result.success(true)
-    } else {
-      return Result.failed(
-        new AppError(
-          DeviceCommunicationError.DisconnectionFailed,
-          `Cannot disconnect device ${this.path}`
-        )
-      )
-    }
-  }
+  // public async disconnect(): Promise<ResultObject<boolean>> {
```

Comment:
to be removed, i guess 🤔

Reply:
fixed 9e2d742a8b4d419f433d38b40c08828f004c23bf